app/views/content/is-teaching-right-for-me/computing.md

```diff
@@ -6,14 +6,15 @@ backlink: "../../"
 subcategory: What to teach
 navigation: 5.43
 navigation_title: Computing
-navigation_description: Discover what it's like to a computing teacher and how you would encourage pupils to learn new digital skills.
+navigation_description: Discover why teaching computing is vital in helping children from all backgrounds to learn the digital skills they'll need throughout life.
```

Comment:
```suggestion
navigation_description: Discover why teaching computing is vital in helping children from all backgrounds to learn the digital skills they'll need throughout life.
```
This is better but feels _slightly_ lengthy for a nav card?

Are we really telling people `why` or are we telling them `how` than could help children learn digital skills?